lib/Common/Memory/PageAllocator.cpp

```diff
@@ -479,6 +462,25 @@ PageSegmentBase<T>::DecommitPages(__in void * address, uint pageCount)
 }
 
 template<typename T>
+void
+PageSegmentBase<T>::PartialDecommitPages(__in void * address, size_t totalPageCount, __in void* addressToDecommit, size_t pageCountToDecommit)
+{
+
+    Assert(address >= this->address);
```

Comment:
done

---
In reply to: [60987860](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60987860) [](ancestors = 60987860)